app/ui/browser/index.jsx

```diff
@@ -97,3 +98,10 @@ ipcRenderer.on('user-agent-service-info', (_, { port, version, host }) =>
 // Set max listeners to Infinity, because each new tab will listen to more
 // events, and we'll intentionally go beyond the 10 listener default.
 ipcRenderer.setMaxListeners(Infinity);
+
+// Add platform as a class on <body> for platform-specific styling. Possible values:
+// 'aix', 'darwin', 'freebsd', 'linux', 'openbsd', 'sunos', 'win32'
+const platform = os.platform();
+const platformClass = platform === 'darwin' ? 'osx' :
+                      platform === 'win32' ? 'win' : 'linux';
```

Comment:
Not sure it's 100% correct to assume third option always linux. Let's go with another conditional branch here and use "other" or something.